src/routes/event.tsx

```diff
@@ -52,44 +54,65 @@ const noMatchesStyle = css`
 const Event = ({ eventKey }: Props) => {
   const matches = useEventMatches(eventKey)
   const eventInfo = useEventInfo(eventKey)
-  const newestIncompleteMatch = matches && nextIncompleteMatch(matches)
-
+  const newestIncompleteMatch = isData(matches) && nextIncompleteMatch(matches)
+  console.log(matches)
   return (
     <Page
       name={eventInfo?.name || <code>{eventKey}</code>}
```

Comment:
This line is where the page title is displayed. Right now, if there is a network error or a non-network error, the page title gets displayed as `Error` or `NetworkError`. Can you change this to use isData so that if there is an error it displays the event key?